### DISCUSSION TITLE
Aiuto sonoff tasmotizzato

### DISCUSSION THREAD

**Discussion by pni87** [indomus/forum#974] [GENERALE - Discussioni generiche]
Buongiorno, vi scrivo per chiedervi un aiuto, ho tasmotizzato un sonoff basic  e modifiicato in contatto pulito per accendere una stufa a pellet collegando in parallelo il tasto di accensione della stufa.
Ho impostato da console di tasmota pulsetime 30 per simulare la pressione del tasto per 3 secondi per accendere. In realtà non funziona perche la mia stufa va in standby, quindi quando il comando viene dato si accene lo schermo della stufa, ma non si accende la stufa stessa, dovrei dargli come un doppio impulso quindi accendere lo schermo e poi tenere premuto i tast di accensione per 3 secondi.
Ho provato diverse cose, ma non sono riuscito perche veramente non ne capisco moltissimo, ho provato a programmare il sonoff da console tramite backlog, facendo una sequenza backlog pulsetime 10; pulsetime 30, in modo da accendere lo schermo e poi con il secondo pulsetime accendere la stufa. Funziona solo la prima  volta, ma devo salvare questo backlog in modo da far funzionare il sonoff in quel modo?. ho provato la funzione blink, 2 lampeggi da 3 secondi in modo che il primo lampeggio mi accende lo schermo il secondo lampeggio mi accende la stufa, anche questo comando scritto in console funziona la prima volta, dopo se provo a fare toggle on  non mi fa la sequenza ma accende e spegne il relay. quindi mi manca qualcosa, ma questi comandi che do devono essere salvati da qualche parte? chi mi aiuta per favore...

**Comment by sky7176** [comment]
Ciao,
perchè non provi con Node-Red?

Da HA dai il comando, poi fai fare a Node-Red la logica per dare due volte l'impulso.

Ma la tua stufa non ha il telecomando IR?
Ti chiedo questo, perchè io la mia l'ho integrata proprio in quel modo, ho preso un Broadlink, gli ho fatto "imparare" i comandi della stufa e poi lo comando via MQTT da Node-Red.

Giuseppe

**Reply by pni87** [reply]
grazie per la risposta, non ha comando ir, ma da applicare inseguito al costo tipo di 200 euro....

**Comment by sky7176** [comment]
Ciao,
allora ricapitolando:
1) il primo impulso attiva il display
2) il secondo accende la stufa

Che intervallo di tempo deve esserci tra i due impulsi per fare in modo che la stufa capisca che deve accendersi?
Ma per spegnere la stufa come fai,ridai i due impulsi,nel senso che se è spenta si accende e se è accesa si spegne?
La mia fa così e per capire se è accesa oppure spenta gli misuro l'assorbimento elettrico.

Io farie in questo modo:
1) creo un unico comando, io uso node-red perchè lo conosco
2) il comando prevede due distinte azioni, la prima immediata che accende il display, la seconda ritartata di n sec. che rimanda il comando e fa accendere la stufa
![Senza nome](https://user-images.githubusercontent.com/13632506/139592138-704d2bbe-8e3f-497c-8175-1933ada02779.png)

Giuseppe

**Reply by snakuzzo** [reply]
@sky7176 ti chiedo cortesemente di rispondere sotto l'ultimo messaggio...senza creare nuovi thread (ho visto che hai risposto "male" anche in altri topic.
Grazie.

**Reply by pni87** [reply]
Si esatto, è come la tua stufa, ok provo a vedere un po con node red, mi sarebbe piaciuto farlo direttamente con tasmota, però credo che non sia possibile, mi scuso se ho risposto "male"

**Reply by sky7176** [reply]
Se hai bisogno chiedi pure.
La "tirata d'orecchie" era per me, nella foga di rispondere a volte non  mi accorgo di quello che combino HI

Giuseppe
